src/components/institution/InstitutionAutocomplete.tsx

```diff
@@ -23,13 +24,16 @@ const InstitutionAutocomplete: FC<InstitutionAutocompleteProps> = ({
   required,
   value = null,
   isLoading = false,
+  id,
   onChange,
-}) => {
+}: InstitutionAutocompleteProps) => {
   const { t } = useTranslation('common');
 
   return (
     <Autocomplete
       {...autocompleteTranslationProps}
+      id={id}
+      aria-labelledby={`${id}-label`}
```

Comment:
hvorfor tenker du at det skal postfixes med `-label`? holder det ikke å bruke `id` som label?

Reply:
Når vi setter id på autocomplete så genererer MUI en `<label id="autocomplete-id-label>`, og for en eller annen grunn må det spesifiseres igjen at Autocompleten er "labelled by" den... Dette fungerer uten å måtte sette den propen på TextField etc :/

Reply:
Åja! Rart 🤔

Reply:
mm, mulig det er sånne ting som gjenstår før den kan gå fra lab til core